tests/config_test.py

```diff
@@ -61,8 +65,10 @@ class TestLoadMinigraph(object):
     def setup_class(cls):
         os.environ['UTILITIES_UNIT_TESTING'] = "1"
         print("SETUP")
+        import config.main
+        imp.reload(config.main)
```

Comment:
why do you need to reload? Not sure I understand the purposes of the reloads below as well. What is changing here so as to reload the config module?

Reply:
This is because of the multi ASIC variables that are set the first time module is loaded. Those variables should have different values based on single or multi ASIC. So reload ensures the test case has the right values.